pandas/core/computation/eval.py

```diff
@@ -198,14 +198,14 @@ def eval(
         <https://docs.python.org/3/reference/simple_stmts.html#simple-statements>`__,
         only Python `expressions
         <https://docs.python.org/3/reference/simple_stmts.html#expression-statements>`__.
-    parser : string, default 'pandas', {'pandas', 'python'}
+    parser : str, default 'pandas', {'pandas', 'python'}
```

Comment:
```suggestion
    parser : {'pandas', 'python'}, optional, default 'pandas'
```

Reply:
The optional is not needed. We just use it when default is `None`, but the `None` means that the parameter is not provided (not the value `None`)